src/views/Peers.tsx

```diff
@@ -154,99 +181,141 @@ export const Peers = () => {
         });
     }
 
+    const setUpdateGroupsVisible = (peerToAction:Peer, status:boolean) => {
+        if (status) {
+            dispatch(peerActions.setPeer({...peerToAction}))
+            dispatch(peerActions.setUpdateGroupsVisible(true))
+            return
+        }
+        dispatch(peerActions.setPeer(null))
+        dispatch(peerActions.setUpdateGroupsVisible(false))
+    }
+
+    const renderPopoverGroups = (label: string, groups:Group[] | string[] | null, peerToAction:PeerDataTable) => {
```

Comment:
Codacy has a fix for the issue: [Replace `····const·renderPopoverGroups·=·(label:·string,·groups:Group[]·|·string[]·|·null,·peerToAction:PeerDataTable` with `··const·renderPopoverGroups·=·(⏎····label:·string,⏎····groups:·Group[]·|·string[]·|·null,⏎····peerToAction:·PeerDataTable⏎··`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
  const renderPopoverGroups = (
```